### DISCUSSION TITLE
使用npu版的paddlepaddle进行检测识别出现的问题

### DISCUSSION THREAD

**Discussion by LegendRGC** [PaddlePaddle/PaddleOCR#12350] [Q&A]
安装完npu版本的paddlepaddle后，使用run_check进行测试能够顺利通过，但是在paddleocr中进行检测识别会出现一些问题。检测阶段耗费时间很长，一张图像需要五六分钟，正常在gpu上只需要几秒即可；识别阶段无法通过，会出现报错，报错结果如图所示。
![微信图片_20240227161619](https://github.com/PaddlePaddle/PaddleOCR/assets/68408391/c503b7f7-f808-41f5-bd33-5102cb3d0aa2)

**Comment by jzhang533** [comment]
duplicated: https://github.com/PaddlePaddle/Paddle/issues/62133
